src/openrct2/drawing/ScrollingText.cpp

```diff
@@ -19,20 +19,15 @@
 #include <algorithm>
 #include <mutex>
 
-#pragma pack(push, 1)
-/* size: 0xA12 */
 struct rct_draw_scroll_text
```

Comment:
I take it this structure is no longer tied to SV6 (or wasn't in the first place)? In that case, it's perhaps better to drop the `rct_` prefix and use TitleCase.

Reply:
It was never tied, just a draw structure.